test_runner/README.md

```diff
@@ -285,6 +285,20 @@ def test_foobar(neon_env_builder: NeonEnvBuilder):
     ...
 ```
 
+The env includes a default tenant and timeline. Therefore, you do not need to create your own
+tenant/timeline for testing.
+
+```python
+def test_foobar2(neon_env_builder: NeonEnvBuilder):
+    env = neon_env_builder.init_start() # Start the environment
+    endpoint = env.endpoints.create_start("main") # Start the compute endpoint
```

Comment:
```suggestion
    with env.endpoints.create_start("main") as endpoint
        # Start the compute endpoint
```

Just some recommendations from @koivunej in my PR. Using the context manager prevents flakiness in the test.

Source: https://github.com/neondatabase/neon/pull/7949#discussion_r1627323976